subsys/bluetooth/audio/has.c

```diff
@@ -589,7 +687,7 @@ static int bt_has_cp_generic_update(struct has_client *client, const struct has_
 	preset_changed_prepare(&buf, BT_HAS_CHANGE_ID_GENERIC_UPDATE, is_last);
 
 	generic_update = net_buf_simple_add(&buf, sizeof(*generic_update));
-	generic_update->prev_index = get_prev_preset_index(preset);
+	generic_update->prev_index = preset_get_prev_index(preset);
```

Comment:
This will provide a wrapped index.  The first item in the list has to have a prevIndex of **0**.